src/www/ui_tests/api/Controllers/GroupControllerTest.php

```diff
@@ -107,35 +108,40 @@ private function getResponseJson($response)
   }
 
 
+  /**
+   * Generate array of groups
+   * @param array $groupIds Group ids to be generated
+   * @return array[]
+   */
+  public function getGroups(): array
+  {
+    $groupIds = [1, 2, 3, 4, 5, 6];
+    $groups = array();
+    foreach ($groupIds as $groupId) {
+      $group = new Group($groupId, null);
+      $groups[] = $group->getArray();
+    }
+    return $groups;
+  }
 
 
   /**
    * @test
-   * -# Test GroupController::deleteGroup() for valid delete request
-   * -# Check if response status is 202
+   * -# Test GroupController::getDeletableGroups()
+   * -# Check if the response is a list of groups.
    */
-  public function testDeleteGroup()
+  public function testGetDeletableGroups()
   {
-    $groupId = 4;
-    $userId = 1;
+    $userId = 2;
+    $groupList = array();
     $this->restHelper->shouldReceive('getUserId')->andReturn($userId);
-    $this->dbHelper->shouldReceive('doesIdExist')
-      ->withArgs(["groups", "group_pk", $groupId])->andReturn(true);
```

Comment:
@dushimsam Looks like during rebase, you removed `testDeleteGroup()` :-|

Reply:
Oh thanks

Reply:
hope things are okay now.